src/datachain/lib/signal_schema.py

```diff
@@ -311,6 +311,10 @@ def mutate(self, args_map: dict) -> "SignalSchema":
                 # renaming existing signal
                 del new_values[value.name]
                 new_values[name] = self.values[value.name]
+            elif name in self.values:
```

Comment:
Is it a part of this PR? Could you please explain why this is needed?

Reply:
I'd suggest avoiding tricks like `mutate(aa=Float)` - it exposes internals to users.

Reply:
About avoiding tricks, I'm not sure I understand what is the real issue there ... by setting `Float` as a type? ... Note that we accept python type instead when changing signal type which is expected behavior for user I would say.